nilearn/plotting/img_plotting.py

```diff
@@ -22,22 +22,22 @@
 from scipy import sparse
 from nibabel.spatialimages import SpatialImage
 
+from ..input_data import NiftiMasker
```

Comment:
It looks like this introduces a circular import problem, so importing leads to the following error:
```shell
>>> from nilearn.plotting import plot_carpet

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/tsalo/Documents/tsalo/nilearn/nilearn/plotting/__init__.py", line 43, in <module>
    from .img_plotting import plot_img, plot_anat, plot_epi, \
  File "/Users/tsalo/Documents/tsalo/nilearn/nilearn/plotting/img_plotting.py", line 25, in <module>
    from ..input_data import NiftiMasker
  File "/Users/tsalo/Documents/tsalo/nilearn/nilearn/input_data/__init__.py", line 5, in <module>
    from .nifti_masker import NiftiMasker
  File "/Users/tsalo/Documents/tsalo/nilearn/nilearn/input_data/nifti_masker.py", line 16, in <module>
    from nilearn.reporting import ReportMixin
  File "/Users/tsalo/Documents/tsalo/nilearn/nilearn/reporting/__init__.py", line 32, in <module>
    from nilearn.reporting.glm_reporter import make_glm_report
  File "/Users/tsalo/Documents/tsalo/nilearn/nilearn/reporting/glm_reporter.py", line 27, in <module>
    from nilearn.plotting import (plot_glass_brain,
ImportError: cannot import name 'plot_glass_brain'
```

Reply:
I think moving the import into the function would fix this, but it's a bit of a hack.

Reply:
Or maybe move that to reporting rather than plotting ?

Reply:
I'll move the import for now, but what if I switched from `NiftiMasker` to `compute_epi_mask`?

Reply:
Thinking a bit further, the circular import tells us that this would probably better fit  in reporting._plot_matrices than here.

Reply:
I was able to fix it without an import within the function by switching from `NiftiMasker` to `compute_epi_mask` and `apply_mask`. Since none of the extra functionality of `NiftiMasker` was being used in the masking stage (i.e., standardization/detrending was done _after_), it seemed like a good choice. That said, do you still want me to move it to `reporting`?

Also, if I move the function to reporting, should I also move the example from "Visualization of brain images" to another section?

Reply:
It might be best to continue this thread in #2488, given its relevance.